pd-configuration-file.md

```diff
@@ -110,6 +110,48 @@ This document only describes parameters that are not included in command-line pa
 
 Configuration items related to pd-server
 
+### `server-memory-limit` <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> This configuration is an experimental feature. It is not recommended to use it in production environments.
+
++ The memory limit ratio for a PD instance. The value `0` means no memory limit.
++ Default value: `0`
++ Minimum value: `0`
++ Maximum value: `0.99`
+
+### `server-memory-limit-gc-trigger` <span class="version-mark">New in v6.6.0</span>
+
+> **Warning:**
+>
+> This configuration is an experimental feature. It is not recommended to use it in production environments.
+
++ The threshold ratio at which PD tries to trigger GC. When the memory usage of PD reaches the value of `server-memory-limit` * `server-memory-limit-gc-trigger`, PD triggers a Golang GC. Only one GC is triggered in one minute.
```

Comment:
```suggestion
+ The threshold ratio at which PD tries to trigger GC. When the memory usage of PD reaches the value of `server-memory-limit * server-memory-limit-gc-trigger`, PD triggers a Golang GC. Only one GC is triggered in one minute.
```

Reply:
Please see [here](https://docs.pingcap.com/tidb/stable/system-variables#tidb_server_memory_limit_gc_trigger-new-in-v640).